### DISCUSSION TITLE
Where is hosted source for Learn section of nextjs.org?

### DISCUSSION THREAD

**Discussion by nrsimha** [vercel/next.js#42476] [Help] [answered]
I have found some bugs in Learn section on nextjs.org and would like to fix it or report it.

What is the place where I can do it?

**Comment by icyJoseph** [comment, marked as answer]
Mmm I think the docs build page, and the learn parts are closed source...

The docs live in this repository https://github.com/vercel/next.js/tree/canary/docs, but only as Markdown files.

The examples in the learn page live here though: https://github.com/vercel/next-learn

**Reply by nrsimha** [reply]
Thank you for info.

Do you think if it will fix issues in Learn text too if issues are fixed in https://github.com/vercel/next-learn ?
There are <a> tags inside of <Link> and Next.js throws error as it seems in new version there should not be there.

**Reply by leerob** [reply]
Yes, please open issues there. Thank you!

**Reply by nrsimha** [reply]
Went to do it now, but found it got already fixed in last commit. :)

**Reply by leerob** [reply]
Should all be patched up now! Woohoo!
